.github/workflows/codespaces-test.yml

```
@@ -9,31 +9,7 @@ on:
-
-      redis:
-        image: redis:7-alpine
-        options: >-
-          --health-cmd "redis-cli ping"
-          --health-interval 10s
-          --health-timeout 5s
-          --health-retries 5
-        ports:
-          - 6379:6379
-
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)